kratos/tensor_adaptors/gauss_point_variable_tensor_adaptor.cpp

```diff
@@ -0,0 +1,133 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//
+
+// System includes
+#include <vector>
+#include <type_traits>
+
+// External includes
+
+// Project includes
+#include "utilities/data_type_traits.h"
+#include "utilities/container_io_utils.h"
+
+// Include base h
+#include "gauss_point_variable_tensor_adaptor.h"
+
+namespace Kratos {
+
+namespace GaussPointVariableTensorAdaptorHelperUtilities
```

Comment:
why do we need those "HelperUtilities"?

Reply:
No, but then i need to put `decltype` gymnastics within the class. I thought, it was not preferred.